upload/modules/CraftingStore/src/Sync/Flow/FullSyncFlow.php

```diff
@@ -45,7 +50,7 @@ public function __construct(
 
     public function performFlow(): bool
     {
-        $serverKey = $this->settingRepository->firstValueByName(SettingEnum::SERVER_KEY, null);
+        $serverKey = $this->settingRepository->firstValueByName(SettingEnum::SERVER_KEY);
```

Comment:
Why did you remove the last value?

Reply:
As PHPStorm suggests `Remove redundant arguments`